uwh-refbox/src/main.rs

```diff
@@ -1497,9 +1703,19 @@ where
 {
     let min = secs / T::from(60u8);
     let sec = secs % T::from(60u8);
-    format!("{:2}:{:02}", min, sec)
+    format!("{:>2}:{:>02}", min, sec)
+}
+
+/*
+
+fn time_panel_center(text: &str) -> std::string::String
+{
+    text_length = text.len();
+    format!("{:^1$}",text, 16 + text_length)
 }
 
+*/
```

Comment:
remove

Reply:
yeah, I was trying to make my own function, didn't finish it, will delete